File: JSBASIC/192_max_product.js
// 192	192_max_product.js	Find the maximum product of any two elements in an array.

let arr = [1, 10, 2, 6, 5, 3];
arr.sort((a, b) => b - a);
let maxProduct = arr[0] * arr[1];
console.log("Array:", arr);
console.log("Maximum product of two elements:", maxProduct);

File: JavaBasic/192_max_product.java
// 192	192_max_product.java	Find the maximum product of any two elements in an array.
import java.util.Arrays;
public class _192_max_product {
    public static void main(String[] args) {
        int[] arr = {1,10,2,6,5,3};
        Arrays.sort(arr);
        int n = arr.length;
        System.out.println("Max product: " + arr[n-1]*arr[n-2]);
    }
}
